xtp/src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+find_package(Python 3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11 REQUIRED)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds PUBLIC binds/include)
```

Comment:
Hmm why public? And it should be Build_interface only unless we install the headers

Reply:
I honestly don't know. Felipe was apparently the engineer who touched this line and he's not working here for a while now. Would you recommend:

```
target_include_directories(xtp_binds PRIVATE $<BUILD_INTERFACE:binds/include>)
```

Reply:
Yes, please

Reply:
done